bindings/wasm/package.json

```diff
@@ -57,21 +59,30 @@
     "copy-webpack-plugin": "^7.0.0",
     "cypress": "^9.3.1",
     "cypress-parallel": "^0.1.8",
+    "fs-extra": "^10.1.0",
     "http-server": "^14.1.0",
     "jsdoc-to-markdown": "^7.1.1",
     "mocha": "^9.2.0",
     "ts-mocha": "^9.0.2",
-    "ts-node": "^10.4.0",
+    "ts-node": "^10.9.1",
     "txm": "^8.0.1",
-    "typescript": "^4.6.2",
+    "typescript": "^4.7.2",
     "wasm-opt": "^1.3.0",
     "wasm-pack": "0.10.1",
     "webpack": "^5.67.0",
     "webpack-cli": "^4.9.2"
   },
   "dependencies": {
+    "@iota/types": "^1.0.0-beta.11",
+    "@types/node-fetch": "^2.6.2",
     "node-fetch": "^2.6.7"
   },
+  "peerDependencies": {
+    "@cycraig/iota-client-wasm": "^0.5.0-alpha.1",
```

Comment:
When or if `iota.rs` allows me to merge in the bindings. I just put it under my account to avoid polluting the global namespace for now, it can be changed to whatever.